encoding/Encoding.java

```diff
@@ -409,7 +411,7 @@ public Infix infix() {
         );
 
         private static final Map<ValueType<?>, Set<ValueType<?>>> ASSIGNABLES = map(
-                pair(OBJECT, set(OBJECT)),
+                pair(OBJECT, set()),
```

Comment:
`OBJECT` valueType never has instances - it's only used for types... conceptually it is not comparable or assignable to anything since it cannot have instances either